romancal/datamodels/schemas/image_files/image.schema.yaml

```diff
@@ -0,0 +1,327 @@
+              type: number
+            crval1:
+              title: first axis value at the reference pixel
+              type: number
+            crval2:
+              title: second axis value at the reference pixel
+              type: number
+            crval3:
+              title: third axis value at the reference pixel
+              type: number
+            ctype1:
+              title: first axis coordinate type
+              type: string
+            ctype2:
+              title: second axis coordinate type
+              type: string
+            ctype3:
+              title: third axis coordinate type
+              type: string
+            cunit1:
+              title: first axis units
+              type: string
+            cunit2:
+              title: second axis units
+              type: string
+            cunit3:
+              title: third axis units
+              type: string
+            cdelt1:
+              title: first axis increment per pixel
+              type: number
+            cdelt2:
+              title: second axis increment per pixel
+              type: number
+            cdelt3:
+              title: third axis increment per pixel
+              type: number
+            pc1_1:
+              title: linear transformation matrix element
+              type: number
+            pc1_2:
+              title: linear transformation matrix element
+              type: number
+            pc1_3:
+              title: linear transformation matrix element
+              type: number
+            pc2_1:
+              title: linear transformation matrix element
+              type: number
+            pc2_2:
+              title: linear transformation matrix element
+              type: number
+            pc2_3:
+              title: linear transformation matrix element
+              type: number
+            pc3_1:
+              title: linear transformation matrix element
+              type: number
+            pc3_2:
+              title: linear transformation matrix element
+              type: number
+            pc3_3:
+              title: linear transformation matrix element
+              type: number
+            ps3_0:
+              title: Coordinate table extension name
+              type: string
+            ps3_1:
+              title: Coordinate table column name
+              type: string
+            s_region:
+              title: spatial extent of the observation
+              type: string
+              default: " "
+            waverange_start:
+              title: lower bound of the default wavelength range
+              type: number
+            waverange_end:
+              title: upper bound of the default wavelength range
+              type: number
+            dispersion_direction:
+              title: Dispersion direction
+              type: integer
+            spectral_order:
+              title: Spectral order number
+              type: integer
```

Comment:
Remove all keywords in wcsinfo above this comment. These are the FITS specific keywords. I'll mark below what else should be removed.

Reply:
Removed.